packages/ember-metal/tests/error_handler_test.js

```diff
@@ -0,0 +1,66 @@
+import Ember from 'ember';
```

Comment:
This test should likely be moved to the `packages/ember/tests` folder, since it requires the global now.